enhancements/update/dont-require-registry-during-boot-and-upgrade.md

```diff
@@ -0,0 +1,201 @@
+---
+title: dont-require-registry-during-boot-and-upgrade
+authors:
+- "@jhernand"
+reviewers:
+- "@avishayt"
+- "@danielerez"
+- "@mrunalp"
+- "@nmagnezi"
+- "@oourfali"
+approvers:
+- "@sdodson"
+- "@zaneb"
+- "@LalatenduMohanty"
```

Comment:
as with the other EP, can we get this down to a single approver?  the other two can be reviewers

Reply:
Yes, who should be that approver?

Reply:
well since you've got 3 candidates listed, i suggest you ask them who amongst them would like to take point and if none of them feel they are the right person hopefully they can make another suggestion.

Reply:
@sdodson @zneb @LalatenduMohanty do you want to be the approver for this?

Reply:
I can not be approver for this enhancement as this does not need any changes in clusterversion object or cluster version operator.

Reply:
Having read through the EP am I correct in summarizing this as a CI testing effort only? We put in place CI jobs that ensure that e2e tests pass without any remote registry access and we're done?

Do we expect a future coordination EP between CVO and PinnedImageSet implementation that would allow something like the PinnedImageSet to assert that a given release's pullspec has been mirrored and then a webhook or pre-apply step could be added to the CVO to go check the PinnedImageSet for it to assert that all nodes have mirrored the desired content?

Reply:
Yes, this is mostly about testing in CI that we can reboot nodes and do upgrades without a registry server. Note that it is not about ensuring that existing end to end tests are able to run without a registry server, rather introducing new tests to verify these specific cases: reboot and upgrade. For example, we may have an end to end test that runs a workload with images that don't exist in the cluster yet. That will require a registry server to pull those images. We are not trying to address that.

Yes, once this and #1481 are accepted we will work on a new EP (based on the now closed #1432) that will introduce the orchestration of an upgrade without a registry server.